htmx_renderer/templates/htmx/list.html

```diff
@@ -11,19 +11,7 @@ <h2 class="row">
       {{meta.name}}
     </span>
     {% if meta.search_enabled %}
```

Comment:
The `if` statement should be inside the partial or else, a search bar will appear on non-searchable models after changing the order for example.
To try is, comment out `search_fields` from any one of the endpoints